client/rpc/src/eth_pubsub.rs

```diff
@@ -160,13 +160,18 @@ impl SubscriptionResult {
 		let mut logs: Vec<Log> = vec![];
 		let mut log_index: u32 = 0;
 		for (receipt_index, receipt) in receipts.into_iter().enumerate() {
+			let receipt_logs = match receipt {
+				ethereum::ReceiptV3::Legacy(d) => d.logs,
```

Comment:
```suggestion
				ethereum::ReceiptV3::Legacy(d) | ethereum::ReceiptV3::EIP2930(d) | ethereum::ReceiptV3::EIP1559(d) => d.logs,
```

Reply:
Yeah, sorry for that. I will change all of them